images/temporal-server/tests/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "version" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME --version"
+}
```

Comment:
> Notes: The helm chart that I guess exists is only inside a GitHub repo and it requires to be cloned, not sure if we can use that
for terraform based testing https://github.com/temporalio/helm-charts

There was a discussion today regarding this - looks like Shivanshu is working on doing the same for another image, maybe check in with him?

Be good to test with the helm chart here if possible

Reply:
Yes, surely I will check and try to see if helm charts are possible, thanks for the review!

Reply:
Added them, feel free to review